api/datastore/backup_test.go

```diff
@@ -36,46 +27,47 @@ func TestCreateBackupFolders(t *testing.T) {
 }
 
 func TestStoreCreation(t *testing.T) {
-	store, teardown := MustNewTestStore(true)
+	_, store, teardown := MustNewTestStore(true)
 	defer teardown()
 
 	if store == nil {
 		t.Error("Expect to create a store")
 	}
 
-	if store.edition() != portainer.PortainerCE {
+	if store.CheckCurrentEdition() != nil {
```

Comment:
We currently don't have the edition field in existing CE databases.  But we should add it I suppose.  Be careful here